infra/modules/service/main.tf

```diff
@@ -286,6 +291,14 @@ resource "aws_cloudwatch_log_group" "service_logs" {
   # TODO(https://github.com/navapbc/template-infra/issues/164) Encrypt with customer managed KMS key
   # checkov:skip=CKV_AWS_158:Encrypt service logs with customer key in future work
 }
+####################
+## Logging Bucket ##
+####################
+
+module "alb_logging" {
+  source         = "../s3-encrypted"
+  s3_bucket_name = var.service_name
+}
```

Comment:
🤔 Won't this create 2 layers of logging? I think this would create:

- an s3 bucket called `var.service_name` that would contain ALB access logs
- an s3 bucket called `${var.service_name}-logging` that would contain logs for the `var.service_name` bucket

Reply:
That's right. It's also why I wanted to rethink my bucket approach to this yesterday. I do think it's a little excessive for this module, but I'm not sure what would be the best way to untangle it from some of the other resources.

Reply:
Sounds good. Let's go ahead and move forward with this PR without this change